keps/sig-cluster-lifecycle/addons/20191013-install-addons-via-kubeadm.md

```diff
@@ -0,0 +1,464 @@
+---
+title: Install Addons via kubeadm
+authors:
+  - "@stealthybox"
+  - "@fabriziopandini"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+reviewers:
+  - TBD
+  - "@dholbach"
+  - "@fabriziopandini"
+  - "@justinsb"
+  - "@neolit123"
+  - "@rosti"
+approvers:
+  - TBD
+  - "@timothysc"
+editor: TBD
+creation-date: 2019-10-13
+last-updated: 2019-10-13
+status: provisional #|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - "/keps/sig-cluster-lifecycle/addons/0035-20190128-addons-via-operators.md"
+  - "/keps/sig-cluster-lifecycle/wgs/0032-create-a-k8s-io-component-repo.md"
+replaces:
+superseded-by:
+---
+
+# Install Addons via kubeadm
+
+## Table of Contents
+<!-- [Tool for generating](https://github.com/ekalinin/github-markdown-toc) -->
+
+- [Install Addons via kubeadm](#install-addons-via-kubeadm)
```

Comment:
k8s-ci-bot says that an error occured at [here](https://github.com/kubernetes/enhancements/pull/1308#issuecomment-564609993) and says that `... 20191013-install-addons-via-kubeadm.md: missing opening TOC tag` in [detailed log](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/enhancements/1308/pull-enhancements-verify/1204792352162451456#1:build-log.txt%3A39)
It was occured on `hack/verify-toc.sh`.
Maybe, aren't `<!-- toc -->` and `<!-- /toc -->`  needed here?

Reply:
Thanks for the guidance on this :+1: